application/core/LSSodium.php

```diff
@@ -124,7 +124,11 @@ public function decrypt($sEncryptedString, $bReturnFalseIfError=false){
 	 * 
 	 * Write encryption key to version.php config file
 	 */
-	protected function generateEncryptionKeys(){			 	
+	protected function generateEncryptionKeys(){
+	if(is_file(APPPATH.'config/security.php')) {
```

Comment:
Wrong indentation?

Reply:
Also, curly brace is not PSR-2 compliant.

Reply:
Since ALL file have wrong identation and are PSR2 broke : i didn'tb update identitention …

Reply:
![Capture d’écran du 2020-05-20 14-23-26](https://user-images.githubusercontent.com/1439428/82445648-7c112080-9aa5-11ea-8096-77412ac5ce7a.png)

Reply:
Related : https://github.com/LimeSurvey/LimeSurvey/pull/1417#discussion_r423551410

Reply:
Done in https://github.com/LimeSurvey/LimeSurvey/pull/1424/commits/b65235b4a2ee0e2c65fd74bb746e3048990fde08 but ONLY on new lines